tests/data/inputData-tests.js

```diff
@@ -59,6 +59,41 @@ test('test that options can be unselected', function (assert) {
   );
 });
 
+test('options can be unselected individually', function (assert) {
+  var options = new Options({
+    data: [
+      {
+        id: 'test',
+        text: 'Test',
+        selected: true
+      },
+      {
+        id: 'test2',
+        text: 'Test2',
+        selected: true
+      },
+      {
+        id: 'test3',
+        text: 'Test3',
+        selected: true
```

Comment:
While setting `selected` here works because of the internals, I'd rather see the `value` set directly on the `<input />` when doing tests (just in case we remove/rename this).